BUILD.gn

```diff
@@ -194,12 +194,15 @@ group("create_dist") {
   deps = []
   if (is_android) {
     deps += [ "build/android:sign_app" ]
-  } else if (is_linux) {
-    # zip will be created along with installer
-    deps += [ ":create_symbols_dist" ]
   } else {
+    deps += [ ":create_symbols_dist" ]
```

Comment:
Is this correct? or should it only be adding for Windows + Linux? (as-is, this would also add for iOS and macOS, which might not be intended)

Reply:
Nothing has been changed.
Before the PR there was a indirect deps to create_symbols_dist via create_dist_zip